server/models/user.py

```diff
@@ -18,3 +18,6 @@ def from_dict(obj: dict) -> 'User':
         registration_started = obj["registration_started"]
         registration_finished = obj["registration_finished"]
         return User(data, registration_started, registration_finished, _id)
+
+    def get_id(self):
```

Comment:
V Pythone sa takto getteri nerobia. 
Funkciu oznacis `@property` a pri pouziti nevolas funkciu ale pristupujes k atributu s nazvom tej funcie.

Reply:
ja by som ani ten getter nerobil len kvôli tomu podčiarovníku na začiatku to pindalo. ale OK, prerobím 👍